src/status_im2/contexts/onboarding/common/background/view.cljs

```diff
@@ -76,8 +76,10 @@
      [carousel/view
       {:animate?          animate?
        :progress          progress
+       :paused            paused
```

Comment:
it's better to use a `?` at the end of boolean variables in Clojurescript. 

i.e `paused?` and `swipeable?`